app/scripts/lib/router.js

```diff
@@ -98,12 +101,14 @@ define(function (require, exports, module) {
       'settings/display_name(/)': createChildViewHandler(DisplayNameView, SettingsView),
       'signin(/)': createViewHandler(SignInView),
       'signin_complete(/)': createViewHandler(ReadyView, { type: 'sign_in' }),
+      'signin_confirmed(/)': createViewHandler(ReadyView, { type: 'sign_in_confirmed' }),
```

Comment:
Does this screen use different text to `signin_complete`? If not, I'd suggest `confirm_sign_in` redirect to `signin_complete`.